daos/events.js

```diff
@@ -0,0 +1,44 @@
+const Events = require('../models/events');
+
+module.exports = {};
+
+module.exports.create = async (name, date, calendarId) => {
+  const event =  await Events.create( {name, date, calendarId});
+  return event;
+};
+
+module.exports.getById = async (id) => {
+  try {
+    const event = await Events.findOne({ _id: id }).lean();
+    return event;
+  } catch (e) {
+    return null;
+  }
+};
+
+module.exports.getAll = async (calId) => {
+  try {
+    const events = await Events.find({calendarId: calId}).lean();
```

Comment:
```suggestion
    const events = await Events.find({ calendarId }).lean();
```
You could then leverage ES6 here